src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -115,6 +115,19 @@
   /**
    * Produces the flag form of a boolean value.
    *
+   * @return {@code [--name=true]} if value=true, {@code [--name=false]} if value=false,
+   *     {@code []} if value=null.
```

Comment:
I still don't understand why we support nulls here. Just use a primitive boolean that is explicitly true or false, never null; per Effective Java Item 49